### DISCUSSION TITLE
Suggestion: add a way to force Rollover

### DISCUSSION THREAD

**Discussion by juliendiabeloop** [tony19/logback-android#248] [Ideas]
Hello,

We use logback-android with RollingFileAppender and SizeAndTimeBasedRollingPolicy but we feel the need to sometimes force the rollover and start a fresh current log file on demand, using the same index incrementation as for a max size overflow. We already prototyped internally an extension of the library to provide this feature.
Is this something you might be interested in including in the official library?

Thanks for reading

**Comment by tony19** [comment]
Yes, sounds interesting. I'm open to PRs for review.
